Implement React frontend + API in Django

ilya90906767 opened the pull request.

greenjune-ship-it commented:
### Restore `node_modules`

I think the whole `node_modules` content can be restored from the `frontend/package-lock.json` file.
We definitely don't want to include dependencies source code into our code base.
It is better to restore the environment once during the build of `frontend/Dockerfile` and then just reuse the container.

Do we have any cons for this?

greenjune-ship-it commented on django_app/django_app/settings.py:
```diff
@@ -31,18 +31,68 @@
+}
+
+from datetime import timedelta
+...
```
We don't need this line :)

greenjune-ship-it commented on docker-compose.yml:
```diff
@@ -1,6 +1,19 @@
+      - NODE_ENV=development
+    depends_on:
+      - django-app
+    command: npm start
```
Can we put it as `CMD ["npm", "start"]` in Dockerfile?
It's better to not overwhelm the compose file with details.

greenjune-ship-it commented on package.json:
Do we need two separate `package.json` files? I think should be enough to have one in the `frontend` folder.

ilya90906767 commented:
> ### Restore `node_modules`
> I think the whole `node_modules` content can be restored from the `frontend/package-lock.json` file. We definitely don't want to include dependencies source code into our code base. It is better to restore the environment once during the build of `frontend/Dockerfile` and then just reuse the container.
> 
> Do we have any cons for this?

I deleted all useless .json files for npm. Now there are only 2 of them. Also i fixed Dockerfile

greenjune-ship-it commented:
Duplicated by PR #3
